olp-cpp-sdk-dataservice-read/tests/CatalogCacheRepositoryTest.cpp

```diff
@@ -65,4 +68,50 @@ TEST(PartitionsCacheRepositoryTest, DefaultExpiry) {
   }
 }
 
+TEST(CatalogCacheRepositoryTest, VersionsList) {
```

Comment:
Please also add a test case for `start_version = -1` as specified by the documentation:

**startVersion**
The beginning of the range of versions you want to get (exclusive). By convention -1 indicates the initial version before the first publication. After the first publication, the catalog version is 0.

Reply:
Test will be added to CatalogRepositoryTest